screens/ProfileScreen.tsx

```diff
@@ -1,25 +1,111 @@
-import { StyleSheet } from "react-native";
+import { AntDesign } from "@expo/vector-icons";
+import { useEffect, useState } from "react";
+import { StyleSheet, TouchableOpacity, ActivityIndicator } from "react-native";
+
 import { Text, View } from "../components/Themed";
+import { UserInfoCard } from "../components/UserInfoCard";
+import { FavoritesCard } from "../components/FavoritesCard";
 import { RootTabScreenProps } from "../types";
+import { useSelector } from "react-redux";
+import { currentReduxUser } from "../redux/signin";
+import { getDocsWithSpecificValue } from "../helper";
+import { Review } from "../Interfaces";
+import { ScrollView } from "react-native-gesture-handler";
+import { ReviewCard } from "../components/ReviewCard";
 
 export default function ProfileScreen({
   navigation,
+  route,
 }: RootTabScreenProps<"Profile">) {
-  return (
-    <View style={styles.container}>
-      <Text>Profile</Text>
-      <View
-        style={styles.separator}
-        lightColor="#eee"
-        darkColor="rgba(255,255,255,0.1)"
-      />
-    </View>
-  );
+  const [follow, setFollow] = useState(false);
+  const user = useSelector(currentReduxUser);
+  const [reviews, setReviews] = useState<Review[]>([]);
+
+  useEffect(() => {
+    getReviews();
+  }, []);
+
+  const getReviews = async () => {
+    try {
+      let data = await getDocsWithSpecificValue(
+        "recensioner",
+        "userID",
+        route.params.id
+      );
+      setReviews(data as Review[]);
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  const toggleButton = () => {
+    setFollow(!follow);
+  };
+
+  if (user) {
```

Comment:
I know I wrote this but since a profile page can be either your own or a random users this if statement should be checking if there is a a user (not just _your_ user as it does now). If you want to visit another users profile you will still only get data from your own user while the url:s params will contain the id for the user you clicked on.

Idea for a solution: 

- Start with comparing route.params.id with 'user' (from redux state). If ids match then this is your profile. Maybe a good idea would be to save that state as a boolean ex _[isMyProfile, setIsMyProfile] = useState<boolean>(false).

- If ids dont match then this is another users profile and then the next step would be do fetch users data from DB and set user state to this user.

 - If isMyProfile = true you should be able to edit stuff like username etc (in a future issue). And also it might be a nice feature to toggle the headings based on user? example: isMyProfile ? "Mina favoriter" : "{user.displayName}s favoriter"